test/com/google/javascript/jscomp/CommandLineRunnerTest.java

```diff
@@ -2152,6 +2152,45 @@ public void testAssumeFunctionWrapper() {
         "function a() {return 'hi'}alert(a);alert(a)");
   }
 
+  public void testWebpackModuleIds() {
+    String inputString =
+        LINE_JOINER.join(
+            "[",
+            "  {\"src\": \"__webpack_require__(2);\", \"path\":\"foo.js\", \"webpackId\": \"1\"},",
```

Comment:
Is there a reason why the webpackId is represented as a string rather than a number?

Reply:
In webpack itself, they are stored as strings. I'm asking "why" on their slack right now. Maybe we can change it.

Reply:
Awesome, nice to catch something in review!